test/unit/enumerable_test.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'set'
+require_relative '../../lib/invoca/utils/enumerable.rb'
+require_relative '../test_helper'
+
+class EnumerableTest < Minitest::Test
+
+  context 'build_hash' do
+    should 'convert arrays of two items to hash key/value pairs' do
+      assert_equal({ 'some' => 4, 'words' => 5 }, ['some', 'short', 'words'].build_hash { |s| [s, s.length] unless s == 'short' })
+    end
+
+    should 'ignore nils returned from map' do
+      assert_equal({ 'some' => 4, 'words' => 5 }, ['some', 'short', 'words'].build_hash { |s|  s == 'short' ? nil : [s, s.length] })
```

Comment:
The above actually tested `nil` also. Probably just remove the `unless` up there so it's the simplest, most common case?

Reply:
Good point.